pkgs/development/python-modules/jax/default.nix

```diff
@@ -89,6 +89,8 @@ buildPythonPackage rec {
     "testKde3"
     "testKde5"
     "testKde6"
+    # Invokes python manually in a subprocess, which does not have the correct dependencies
```

Comment:
@samuela I don't think figuring out upstream's subprocess/pythonpath pollution issues is a necessity for us. Shall we let this through?

Reply:
ok fair enough, fine with me